packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -519,6 +514,7 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
   /* @conditional-compile-remove(unsupported-browser) */
   public allowUnsupportedBrowserVersion(): void {
     this.context.setAllowedUnsupportedBrowser();
+    this.context.updateClientState(this.callClient.getState());
```

Comment:
remove this line

Reply:
Lets not block PR on this, filed a bug instead [Bug 3093292](https://skype.visualstudio.com/SPOOL/_workitems/edit/3093292): [Web] Remove unsupportedBrowsers continueAnyway use of updateClientState